### ISSUE TITLE
genomes of pecan and Chinese hickory

### ISSUE THREAD

**Issue by mestato** [statonlab/hardwoods_site#521]
### Publication and Data Information
https://academic.oup.com/gigascience/article/8/5/giz036/5484800

### Checklist
See [New Genome Documentation](https://github.com/mestato/statonlabprivate/wiki/How-to-load-a-new-organism-and-genome) for detailed instructions.
- [x] Create Organism
- [x] Create Publication
- [x] Create Reference Genome
- [x] Create InterProScan Annotation
- [x] Create BLAST Annotation
- [x] Run Chado FASTA Loader for CDS
- [x] Run Chado FASTA Loader for Polypeptides
- [x] Publish Tripal content
- [x] Create KEGG Annotation
- [x] Run Chado KEGG Loader
- [x] Run Chado BLAST XML results loader
- [x] Run Chado InterProScan XML results loader
- [x] Create Blast Database

**Comment by patricksis** [comment]
I was looking through the GeneBank to find these organism, couldn't find them. Maybe I'm looking in the wrong directory @mestato 
[ftp://ftp.ncbi.nlm.nih.gov/genomes/genbank/plant/](url)

**Comment by mestato** [comment]
I was a reviewer of the paper so I know its available somewhere....
try this site:
http://gigadb.org/dataset/100571

**Comment by patricksis** [comment]
Ah I should've checked Giga, thank you

**Comment by patricksis** [comment]
**Carya illinoinensis (Pecan)**
Organism:
https://www.hardwoodgenomics.org/organism/Carya/illinoinensis
Publication:
https://www.hardwoodgenomics.org/Publication/3156255
Reference Genome:
https://www.hardwoodgenomics.org/Genome-assembly/3156256
Interproscan annotation:
https://www.hardwoodgenomics.org/InterProScan-annotation/3156261
Swissprot annotation:
https://www.hardwoodgenomics.org/BLAST-annotation/3156263
Trembl annotation:
https://www.hardwoodgenomics.org/BLAST-annotation/3156264
Fasta cds job:
https://www.hardwoodgenomics.org/admin/tripal/tripal_jobs/view/777293
Fasta peptide job:
https://www.hardwoodgenomics.org/admin/tripal/tripal_jobs/view/777297
Trembl loader:
https://www.hardwoodgenomics.org/admin/tripal/tripal_jobs/view/779425
Swissprot loader:
https://www.hardwoodgenomics.org/admin/tripal/tripal_jobs/view/779430
Publish Tripal content:
https://www.hardwoodgenomics.org/admin/tripal/tripal_jobs/view/779434
IPS loader:
https://www.hardwoodgenomics.org/admin/tripal/tripal_jobs/view/779439
KEGG annotation:
https://www.hardwoodgenomics.org/KEGGresults/3220248
kEGG loader:
https://www.hardwoodgenomics.org/admin/tripal/tripal_jobs/view/781698

**Comment by patricksis** [comment]
**Carya cathayensis (Chinese Hickory)**
Organism:
https://www.hardwoodgenomics.org/organism/Carya/cathayensis
Publication:
https://www.hardwoodgenomics.org/Publication/3156255
Reference Genome:
https://www.hardwoodgenomics.org/Genome-assembly/3156260
InterProScan annotation:
https://www.hardwoodgenomics.org/InterProScan-annotation/3156262
Swissprot annotation:
https://www.hardwoodgenomics.org/BLAST-annotation/3156265
Trembl annotation:
https://www.hardwoodgenomics.org/BLAST-annotation/3156266
Cds fasta upload:
https://www.hardwoodgenomics.org/admin/tripal/tripal_jobs/view/777455
Peptide fasta upload:
https://www.hardwoodgenomics.org/admin/tripal/tripal_jobs/view/777459
Publish Tripal content:
https://www.hardwoodgenomics.org/admin/tripal/tripal_jobs/view/779440
Swissprot loader:
https://www.hardwoodgenomics.org/admin/tripal/tripal_jobs/view/779699
Trembl loader:
https://www.hardwoodgenomics.org/admin/tripal/tripal_jobs/view/781684
KEGG annotation:
https://www.hardwoodgenomics.org/KEGGresults/3220249
KEGG loader:
https://www.hardwoodgenomics.org/admin/tripal/tripal_jobs/view/784574

**Comment by patricksis** [comment]
@almasaeed2010 got this error when trying to create the second genome
```Could not save the TripalEntity: Unable to write fields to Chado: Array ( [schema__publication] => Array ( [0] => Array ( [value] => 41 [chado-analysis_pub__analysis_pub_id] => [chado-analysis_pub__analysis_id] => [chado-analysis_pub__pub_id] => 41 [pub_title] => The genomes of pecan and Chinese hickory provide insights into Carya evolution and nut nutrition [id:41] ) ) [local__source_data] => Array ( [0] => Array ( [value] => [chado-analysis__sourcename] => [chado-analysis__sourceversion] => __NULL__ [chado-analysis__sourceuri] => __NULL__ [source_data] => Array ( [sourcename] => [sourceversion] => [sourceuri] => ) ) ) [sio__annotation] => Array ( [0] => Array ( [value] => delete_me [chado-analysis_cvterm__analysis_cvterm_id] => [chado-analysis_cvterm__cvterm_id] => [chado-analysis_cvterm__analysis_id] => [cv_id] => 0 [cvterm_name] => [chado-analysis_cvterm__is_not] => [chado-analysis_cvterm__rank] => [chado-analysis_cvterm_is_not] => ) ) [sbo__database_cross_reference] => Array ( [0] => Array ( [value] => delete_me [chado-analysis_dbxref__analysis_dbxref_id] => [chado-analysis_dbxref__analysis_id] => [chado-analysis_dbxref__dbxref_id] => [db_id] => 0 [accession] => ) ) [field_number_of_scaffolds] => Array ( [0 Referesis_id] => [chado-analysisprop__value] => [chado-analysisprop__type_id] => [chado-analysisprop__rank] => ) ) [local__time_executed] => Array ( [0] => Array ( [value] => 2019-05-16 19:02:00 [show_todate] => 1 [timezone] => America/Halifax [offset] => -10800 [offset2] => -10800 [value2] => 2019-05-16 19:02:00 ) ) )```
```Cannot save entity```

Maybe I should differentiate them somehow?

**Comment by almasaeed2010** [comment]
Yeah they need to have a different version, program or type. The name alone doesn't count.

**Comment by patricksis** [comment]
For both of these organism, IPS results aren't showing up. 
Could someone take a look at this when they get a chance @mestato @almasaeed2010 ?

**Comment by mestato** [comment]
@almasaeed2010 needs to see the files, could you transfer them back over so he can take a look?

**Comment by patricksis** [comment]
All IPS files for Carya illinoinensis and Carya cathayensis have been added back to hardwoods.
The paths are ```/var/www/html/sites/default/files/sequences/Carya_cathayensis/IPS/xmls```
and ```/var/www/html/sites/default/files/sequences/Carya_illinoinensis/IPS```

**Comment by almasaeed2010** [comment]
Thanks @patricksis I’ll look at them when I get a minute

**Comment by almasaeed2010** [comment]
Files look fine. I submitted a new job without using any regular expressions to test. 

https://hardwoodgenomics.org/admin/tripal/tripal_jobs/view/790791

**Comment by almasaeed2010** [comment]
The job worked. Probably was a simple mistake in the form somewhere.

**Comment by patricksis** [comment]
Thanks @almasaeed2010 I'll submit a new job for Carya illinoinensis

**Comment by patricksis** [comment]
Carya illinoinensis redo IPS:
https://www.hardwoodgenomics.org/admin/tripal/tripal_jobs/view/791195

**Comment by patricksis** [comment]
@cricha59 Can you look at these two organisms when you get the chance? IPS results might not show up, if they don't just reload the cache.

**Comment by mestato** [comment]
Reopening due to errors. On genome assembly page, 
- "-All gene models genome" should be "Genome scaffolds" (both species)
- That link doesn't work and needs to be updated (cathayensis only)
- the downloads need to use our typical HTML for downloadable files (both species)

For an example of how to structure download links please see https://hardwoodgenomics.org/Genome-assembly/2209433?tripal_pane=group_summary_tripalpane

**Comment by patricksis** [comment]
All links have been replaced and fixed with HTML format, as well as the labeling.
